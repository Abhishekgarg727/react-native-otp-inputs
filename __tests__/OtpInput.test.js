describe('<OtpInput />', () => {
  test('should test something', () => {})
})
