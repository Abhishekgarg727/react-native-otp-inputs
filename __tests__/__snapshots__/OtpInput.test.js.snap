// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OtpInput /> render with default props 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#a1a1a1",
        "borderRadius": 6,
        "borderWidth": 1,
        "height": 53,
        "margin": 10,
      },
      undefined,
      Object {
        "borderColor": undefined,
      },
    ]
  }
>
  <TextInput
    allowFontScaling={true}
    onBlur={[Function]}
    onChange={[MockFunction]}
    onFocus={[Function]}
    onKeyPress={[MockFunction]}
    rejectResponderTermination={true}
    style={
      Array [
        Object {
          "color": "#ffffff",
          "fontSize": 24,
          "paddingTop": 10,
          "textAlign": "center",
          "width": 40,
        },
        undefined,
        undefined,
      ]
    }
    underlineColorAndroid="transparent"
  />
</View>
`;

exports[`<OtpInput /> render with error message 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#a1a1a1",
        "borderRadius": 6,
        "borderWidth": 1,
        "height": 53,
        "margin": 10,
      },
      undefined,
      Object {
        "borderColor": undefined,
      },
    ]
  }
>
  <TextInput
    allowFontScaling={true}
    onBlur={[Function]}
    onChange={[MockFunction]}
    onFocus={[Function]}
    onKeyPress={[MockFunction]}
    rejectResponderTermination={true}
    style={
      Array [
        Object {
          "color": "#ffffff",
          "fontSize": 24,
          "paddingTop": 10,
          "textAlign": "center",
          "width": 40,
        },
        undefined,
        Object {
          "color": "#00ff00",
        },
      ]
    }
    underlineColorAndroid="transparent"
  />
</View>
`;
